### ISSUE TITLE
Select: ARIA hidden element must not be focusable or contain focusable elements

### ISSUE THREAD

**Issue by Josef-Kainz-Reichardt** [primefaces/primevue#6103]
### Describe the bug

The `Select` componen (former `Dropdown`) contains a static `<span>`  element in the dropdown list which breaks accessibility:
![grafik](https://github.com/user-attachments/assets/e7470265-bb98-4095-89bd-7bbfed9047fd)

### Reproducer

https://primevue.org/select/

### PrimeVue version

3 & 4

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vite

### Browser(s)

Firefox, Chrome, ... all

### Steps to reproduce the behavior

1. Install Axe browser plugin (unfortunatelly Google Chrome only): https://www.deque.com/axe/
2. Go to https://primevue.org/select/
3. Open the browser dev tool and go to the tab "axe DevTools"
4. Open a dropdown on the primevue demo page
5. Click on "Full Page Scan" in the axe panel (while the dropdown is still open)

### Expected behavior

The expected behavior is that axe doesn't find any issue on the primevue demo page.

**Comment by LanFeusT23** [comment]
This happens on Menu buttons too fyi

**Comment by bonniegrubbs** [comment]
Agreed, I'm seeing this on the menu component as well. To replicate, see the Primevue example [here](https://tailwind.primevue.org/menu/#router)
<img src="https://github.com/user-attachments/assets/96d270ab-d701-4956-990d-de28e7c111b7" width="600" />

**Comment by scottmillion** [comment]
Had this issue on TieredMenu component. 

@mousedown.prevent fixed it.

**Comment by tugcekucukoglu** [comment]
This issue will be fixed in version 5.0.2. Thank you.
